kube/values.go

```diff
@@ -9,15 +9,15 @@ import (
 )
 
 // MakeValues returns a Mapping with all default values for the Helm chart
-func MakeValues(roleManifest *model.RoleManifest, defaults map[string]string) (helm.Node, error) {
+func MakeValues(settings *ExportSettings) (helm.Node, error) {
```

Comment:
The way this is being used, it might as well be `func (settings *ExportSettings) MakeValues() (helm.Node, error)`…

Reply:
Yeah, but really only if we turn all the functions into settings methods.  Is that what you are suggesting here?

Reply:
Less suggesting and more wishing the world was that way in the first place. I don't think it'd make sense to change it in this PR. 😢